cpp/README.md

```diff
@@ -32,6 +32,7 @@ Building Arrow requires:
 * A C++11-enabled compiler. On Linux, gcc 4.8 and higher should be sufficient.
 * CMake 3.2 or higher
 * Boost
+* Bison/flex (for parquet only)
```

Comment:
These are only required for building Apache Thrift from source. Can you reflect that here?

Reply:
reworded slightly.